sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_utils/__init__.py

```diff
@@ -43,6 +44,15 @@ def _get_customer_ikey_from_env_var():
     return _CUSTOMER_IKEY_ENV_VAR
 
 
+# TODO: Add environment variable to enable/disable diagnostics
+def _is_diagnostics_enabled():
+    if _is_on_functions():
```

Comment:
Can probably add a TODO to change this logic to just use "is_attach_enabled" once we have functions attach.

Reply:
Not sure I understand. Will sync